pipeline/utils/task_utils.py

```diff
@@ -36,6 +36,13 @@ def gradle_task(toolkit_path, task_name, task_args):
             task_name, '-Pclargs=' + ','.join(task_args)]
 
 
+def api_name(short_name, version, is_cloud):
```

Comment:
Ah, I don't think I realized there was going to be more of a distinction beyond cloud/non-cloud. Changed this value to "organization_name".